packages/deployment/upgrade-test/Makefile

```diff
@@ -10,10 +11,10 @@ local_sdk:
 	(cd ../ && make docker-build-sdk)
 
 build:
-	docker build --build-arg BOOTSTRAP_MODE=main --progress=plain $(buildTargetFlag) -t $(REPOSITORY):$(dockerLabel) -f Dockerfile upgrade-test-scripts
+	docker build --build-arg BOOTSTRAP_MODE=main --build-arg DEST_IMAGE=$(DEST_IMAGE) --progress=plain $(buildTargetFlag) -t $(REPOSITORY):$(dockerLabel) -f Dockerfile upgrade-test-scripts
```

Comment:
Would we prefer something like `build_local` that uses `ghcr.io/agoric/agoric-sdk:dev` to avoid having to set `DEST_IMAGE=ghcr.io/agoric/agoric-sdk:latest` when running make build?

Reply:
sgtm